certificate-service/domain/src/main/java/se/inera/intyg/certificateservice/domain/certificate/model/Certificate.java

```diff
@@ -196,5 +199,29 @@ public void send(ActionEvaluation actionEvaluation) {
         .sentAt(LocalDateTime.now(ZoneId.systemDefault()))
         .build();
   }
+
+  public void revoke(ActionEvaluation actionEvaluation, String revokeReason, String revokeMessage) {
```

Comment:
Good if we work with domain objects. Use RevokedInformation instead of two string arguments.